internal/events/eventer.go

```diff
@@ -98,7 +100,7 @@ var _ message.Publisher = (*Eventer)(nil)
 
 // Setup creates an Eventer object which isolates the watermill setup code
 // TODO: pass in logger
-func Setup(ctx context.Context, cfg *config.EventConfig) (*Eventer, error) {
+func Setup(ctx context.Context, cfg *config.EventConfig, db *sql.DB) (*Eventer, error) {
```

Comment:
Instead of passing the `db` as a new argument, shouldn't we instead just rely on the configuration given within `cfg` and do the database connection instantiation as part of the builder?